R/splitBy.R

```diff
@@ -0,0 +1,181 @@
+#' Split \code{SummarizedExperiment} by grouping variables
+#'
+#' @param x A
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
+#'   object
+#'
+#' @param grouping A single character value for selecting the grouping variable
+#'   from \code{colData} or \code{rowData}.
+#' 
+#' @param direction A single character value for selecting from where grouping
+#'   variable should be searched. Must be "col" or "row". 
```

Comment:
if (?) MARGIN is used as the argument name, we could also allow 1 (row) and 2 (col) as this is standard.